reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -138,7 +140,11 @@ data class EvaluatedModel(
             propertyNamingStrategy = PROPERTY_NAMING_STRATEGY
         }
 
-        private val JSON_MAPPER = JsonMapper().apply(MAPPER_CONFIG)
+        private val JSON_MAPPER by lazy {
+            val factory = JsonFactory()
+            factory.disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
```

Comment:
These could be a one-liner:

    val factory = JsonFactory().disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)